eox_tenant/models.py

```diff
@@ -65,6 +66,99 @@ def get_microsite_for_domain(cls, domain):
         return microsites[0] if microsites else None
 
 
+class TenantConfigManager(models.Manager):
+    """
+    Custom managaer for Tenant Config model.
+    """
+    def get_configurations(self, domain):
+        """
+        Execute optimized query to get site configurations.
+        """
+        from django.db import connection
+
+        with connection.cursor() as cursor:
+            cursor.execute("""
```

Comment:
This with django queryset tools was an expensive operation?

Reply:
Yes, basically this is an optimized query based on https://www.wix.engineering/post/scaling-to-100m-mysql-is-a-better-nosql